src/components/Summary/CosignSummary.js

```diff
@@ -73,7 +73,7 @@ const CosignSummary = ({
     await destroySession();
   };
 
-  const [sessionExpired, expiryDate] = useSessionTimeout(async () => {
+  const [expiryDate] = useSessionTimeout(async () => {
```

Comment:
the signature of `useSessionTimeout` is now not used consistently - the first argument in the return tuple is the `expired` boolean.

In `src/components/Form.js` you do account for this with `[, arg2, arg3]`, but this one has a type issue.

If the `expired` value is not relevant (anymore), I prefer updating the `useSessionTimeout` return value to return a two-tuple rather than the three-tuple it has now.